DXMainClient/Startup.cs

```diff
@@ -153,6 +155,25 @@ public void Execute()
                 fhc.CalculateHashes();
             }
 #endif
+            // Load native steam binaries.
+            SteamNative.Initialize();
```

Comment:
Also, does the Steam SDK work (or at least doesn't crash the client) in Linux? It's better to test it. If not (I don't know whether it works or not), an if statement should be introduced to skip the Steam API if the OS is not Windows.